src/fides/api/api/v1/endpoints/user_endpoints.py

```diff
@@ -151,7 +151,7 @@ def update_user_password(
     """
     _validate_current_user(user_id, current_user)
 
-    if not current_user.credentials_valid(
+    if not current_user.authorization.credentials_valid(
```

Comment:
this looks like a find/replace mistake, going to correct!

Reply:
addressed all of these instances in 18c493c4c4764f59b39fd785544ae95ca27996db

Reply:
oof, yep. unfortunate

Thank you for the catch and fix!